spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1651,6 +1652,13 @@ def _do_mapping(self, run_time, n_machine_time_steps, total_run_time):
                     "Reports", "write_network_specification_report")):
                 algorithms.append("NetworkSpecificationReport")
 
+        # write data in speed up report if required
```

Comment:
Why not just make this a simple assignment?

    inputs["WriteDataInSpeedUpReportFlag"] = self._config.getboolean("Reports", "write_data_in_speed_up_report")

You're gaining nothing from doing it the more complicated way.

Reply:
fixed

Reply:
good